plugins/textpattern.md

```diff
@@ -101,8 +102,11 @@ The block patterns do not have an `end` property. This allows for patterns to be
 
 ```js
 tinymce.init({
-  selector: 'textarea',  // change this value according to your HTML
-  plugin: 'textpattern lists',
+  selector: 'textarea', // change this value according to your HTML
+  // make sure to have the required plugins here,
```

Comment:
This is mentioned a few times now, I wonder whether it might be a bit of an overload here?

Reply:
Maybe
```
  // The `lists` plugin is required for list-related text patterns
```
And delete the rest of the comment?

Reply:
Any more thoughts on this one, please?

Reply:
Done